styles.css

```diff
@@ -772,3 +790,134 @@ button {
 }
 
 /* #endregion */
+
+.details-cont {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #c1c7d0;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 2;
+}
```

Comment:
- [ ] Great work fixing the positioning of the buttons as stated by the previous reviewer, however, there is still a bit to be done. Kindly ensure that the buttons are the same size as in the Figma design so that it does not overlap with the text as it currently does. Also, please ensure that the cancel button is the same size as the one in the design.

### Your design
![Screenshot 2023-06-21 at 21 41 49 (2)](https://github.com/s2jack/create-portfolio-website1/assets/64774936/5e13de1b-7f4b-4fac-93f5-1bef292c78f9)

### Figma design
![](https://user-images.githubusercontent.com/107632780/247614645-5c0155fe-e191-4f4b-af19-917457e33071.png)

Reply:
hello @Ghiftee , Thanks for your time. I want to ask you about your screen properties. Because I can't see any issue like that on my side. I launched my project on the browser, then I change screen sizes with developer tools. But I didn't get any error like that. Kindly ask you give me your screen sizes that you took these issues. It would be very helpful to figure it out.